src/start.ts

```diff
@@ -5,6 +5,7 @@ import "missing-native-js-functions";
 import { config } from "dotenv";
 config();
 import { FosscordServer } from "./Server";
+console.log(process.env.MONGO_URL)
```

Comment:
Please remove this as it exposes the connection string. We already have a logger that strips the credentials

Reply:
Oh yeah, I forgot to remove that